src/Components/FilteredBeast.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Form from 'react-bootstrap/Form';
+
+class FilteredBeast extends React.Component {
+  render() {
+    return (
+      <div>
+        <Form.Select
+          aria-label='Default select example'
+          onChange={this.props.handleChange}
+        >
+          <option value='default'>All Beasts</option>
```

Comment:
Change your value here to another number. '0' works well. The reason to do this is because you're going to parseInt your selection, and you can't parseInt 'default'. It make it easier to compare values if everything is the same data type.